languages/komoju-woocommerce.pot

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: komoju-woocommerce\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-06-03 01:40+0000\n"
+"POT-Creation-Date: 2022-06-03 11:08+0900\n"
```

Comment:
This whole file is automatically generated - no need to pour over each and every line :wink: